Modification des dépendances d'installation

seigneurfuo opened the pull request:
Modification des instructions d'installation pour les dépendances pour éviter d'installer toute la pile Xorg.

La version allégée de Debian pour la rasperry-pi viens sans installation de la pile graphique Xorg.

Cependant l'installation du paquet unique `erlang` installe de très nombreuses dépendances indirectes dont toute la pile graphique Xorg.


Cette pull-request permet de corriger le problème en installant seulement les paquets `erlang-nox erlang-dev` afin de garder un système de base léger.

seigneurfuo committed:
Modification des dépendances d'installation pour éviter l'installation de toute la pile Xorg.

pguyot commented:
Merci, bonne idée.
Est-ce que vous pouvez en profiter pour modifier les workflows GH action de la même manière ?
Aussi je ne suis pas sûr que erlang-dev soit nécessaire. On pourrait vérifier avec la CI.

seigneurfuo commented:
Lors de mes test, j'avais juste installé ```erlang-nox```.
J'ai eu une erreur de compilation car un fichier n'était pas trouvé: ```erl_nif.h```
J'ai résolu le soucis en installant ```erlang-dev```: [Paquets Debian contenant erl_nif.h](https://packages.debian.org/search?searchon=contents&keywords=erl_nif.h&mode=path&suite=stable&arch=any)

seigneurfuo committed:
Update build-pynab.yml

Modification des actions Github pour modifier les paquets d'installation erlang-nox et erlang-dev

seigneurfuo committed:
Update build-raspios.yml

Modification des actions Github pour modifier les paquets d'installation erlang-nox et erlang-dev
